addons/loyalty/models/loyalty_history.py

```diff
@@ -1,23 +1,41 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from datetime import datetime, timedelta
+
+from odoo import api, fields, models
+from odoo.tools import float_round
 
 
 class LoyaltyHistory(models.Model):
     _name = 'loyalty.history'
     _description = "History for Loyalty cards and Ewallets"
     _order = 'id desc'
 
-    card_id = fields.Many2one(comodel_name='loyalty.card', required=True, index=True, ondelete='cascade')
+    card_id = fields.Many2one(
+        comodel_name='loyalty.card',
+        required=True,
+        index=True,
+        ondelete='cascade',
+        readonly=True,
+    )
     company_id = fields.Many2one(related='card_id.company_id')
 
-    description = fields.Text(required=True)
+    description = fields.Text(required=True, readonly=True)
```

Comment:
Salesmen still need to be able to set a custom description when gifting points to customers